src/module/utils/actorDamage.ts

```diff
@@ -101,8 +103,16 @@ export class Stats {
     return retValue;
   }
 
+  totalDamage(): number {
+    const totalDamage = Roll.safeEval(this.damageFormula.replaceAll("dmg",this.damageValue).replaceAll("armor",this.effectiveArmor));
```

Comment:
Thanks! A couple of questions here.  Do we need to expose more of the actor data here - e.g. characteristics?  I'm just wondering whether the formula needs to be more general and then use full Roll.eval.  Also, maybe put @ in front of variables @dmg and @effectiveArmor to denote to users that these are variables?  That is the format for most of the other input formulas.  Spell out dmg as damage.  Unfortunately we can't localize.  Please let me know your thoughts.

Reply:
1. Changeing to @effectiveArmor and @damage is no problem.
2. The useage of safeEval here was to prevent users from input something that has a real roll in it(p.e. 2D+@damage) because the evaluation might be done multiple times with then different roll results.
3. But adding access to characteristics might be a good idea.
(changed to the same logic that encumbrance currently used)

Reply:
Great...I think we should make it a world setting so users can't change it - only GMs. I'll also add support for auto complete before merging.